make_hh_acs_data.R

```diff
@@ -836,58 +718,140 @@ get_acs_hs <- function(year) {
+               ) |>
+  update_field("fraction_conditions",
+               title = "Fraction of Housing Units with Substandard Housing Conditions",
+               description = "substandard housing: incomplete plumbing or kitchens, overcrowding, 30% or more of household income spent on rent or owner costs"
+               ) |>
+  update_field("fraction_builtbf1970",
+               title = "Fraction of Housing Units Built Before 1970"
+               ) |>
+  update_field("fraction_vacant",
+               title = "Fraction of Housing Units that are Vacant"
+               ) |>
+  update_field("fraction_nhl",
+               title = "Fraction of People Not Hispanic/Latino"
+               ) |>
+  update_field("fraction_nhl_w",
+               title = "Fraction of People White and Not Hispanic/Latino"
+               ) |>
+  update_field("fraction_nhl_b",
+               title = "Fraction of People Black and Not Hispanic/Latino"
+               ) |>
+  update_field("fraction_nhl_o",
+               title = "Fraction of People Not Black, Not White, and Not Hispanic/Latino"
+               ) |>
+  update_field("fraction_hl",
+               title = "Fraction of People Hispanic/Latino"
+               ) |>
+  update_field("fraction_hl_w",
+               title = "Fraction of People White and Hispanic/Latino"
+               ) |>
+  update_field("fraction_hl_b",
+               title = "Fraction of People Black and Hispanic/Latino"
+               ) |>
+  update_field("fraction_hl_o",
+               title = "Fraction of People Not Black, Not White, and Hispanic/Latino"
+               ) |>
+  update_field("fraction_lesh",
+               title = "Fraction of Households Speaking Limited English",
+               description = "Available from 2016 onwards"
+               ) |>
+  update_field("median_income",
+               title = "Median Household Income"
+               ) |>
+  update_field("median_income_2010adj",
+               title = "Median Household Income (in 2010 USD)"
+               ) |>
+  update_field("fraction_hs",
+               title = "Fraction of Adults with At Least High School Education",
+               description = "Available from 2012 onwards"
+               )
 
 # save to disk
-write_tdr_csv(d)
-
-# save metadata to md file
-options(knitr.kable.NA = "")
-cat("#### Metadata\n\n", file = "metadata.md", append = FALSE)
-codec::glimpse_attr(d) |>
-  knitr::kable() |>
-  cat(file = "metadata.md", sep = "\n", append = TRUE)
-cat("\n#### Schema\n\n", file = "metadata.md", append = TRUE)
-cat("Columns ending with `_moe` represent the margin of error accompanying another column and are not included in the schema table here.\n\n", file = "metadata.md", append = TRUE)
-d |>
-  dplyr::select(-ends_with("moe")) |>
-  codec::glimpse_schema() |>
-  knitr::kable() |>
-  cat(file = "metadata.md", sep = "\n", append = TRUE)
+fr::write_fr_tdr(d, dir = here::here())
```

Comment:
why are you specifying the output dir here?   it will write by default to the working directory, right?

Reply:
I was getting this error 

```
Error in fr::write_fr_tdr(d) : argument "dir" is missing, with no default
```

Reply:
I forgot I made that no default! use `fr::write_fr_tdr(d, '.')` instead to avoid using another package.  I would usually use fs::path_package() so that it writes to the correct place no matter where it runs, but this isn't a package, so that won't work.